src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Alerts and AI transition guide 2022
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - EOL
+  - Transition guide
+metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+---
+
+Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
+
+## What's changing [#changes]
+
+Here is a summary of what we plan to EOL in 2022, why this is happening, and what will replace the current experience for Alerts and Applied Intelligence.
+
+If you have questions or comments about anything in this EOL transition guide, don't hesitate to:
+
+* Reach out to your New Relic account representative.
+* Use the reply thread in our [Explorers Hub post](https://discuss.newrelic.com/t/big-news-for-alerts-and-applied-intelligence/161436).
+
+<CollapserGroup>
+
+  <Collapser
+    id="outlier"
+    title="February 2022: NRQL baselines to replace outlier NRQL alert condition types."
+  >
+
+  **Reason for EOL:** Inefficient cluster identification and analysis capability.
+
+  **Replacement:** Use NRQL baselines to watch for anomalies in your cluster or to group similar signals. This method is an improvement over outlier conditions, since it will alert you to exactly which member of the group is misbehaving. You may be able to meet the same need as your old outlier conditions met by using a faceted baseline condition.
```

Comment:
```suggestion
  **Replacement:** Use NRQL baselines to watch for anomalies in your cluster or to group similar signals. This method is an improvement over outlier conditions, since it will alert you to exactly which member of the group is misbehaving. You may be able to meet the same need as your old outlier conditions by using a faceted baseline condition.
```